.travis.yml

```diff
@@ -18,6 +18,7 @@ install:
 - source activate deepchem
 - pip install yapf==0.16.0
 - python setup.py install
+- KERAS_BACKEND=tensorflow
```

Comment:
Removed -- I needed to install keras from the anaconda repo.  It was installing an old one on travis because we didn't explicitly pin it (0.3.x).  I also updated the conda version used by travis as it was old, and didn't install the correct keras by default.